.github/workflows/issues.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: GH issues
+on:
+  workflow_call:
+    inputs:
+      ado_area_path:
+        description: 'ADO area path where ADO tickets should be created'
+        required: true
+        default: "2DegreesInvesting\\Test Team 1"
+        type: string
+      do-gh-to-ado:
+        description: 'Flag to run `gh-to-ado`'
+        required: false
+        default: true
+        type: boolean
+    secrets:
+      GH_TOKEN:
```

Comment:
you should be able to use the built-in `${{ secrets.GITHUB_TOKEN }}` or `${{ GitHub.token }}` (equivalent calls) rather than explicitly passing it.

Reply:
So I don't need to specify it at all?